backend/pkg/rest/apiinventory_controller.go

```diff
@@ -26,6 +26,17 @@ import (
 	_database "github.com/apiclarity/apiclarity/backend/pkg/database"
 )
 
+func (s *Server) PostAPIInventory(params operations.PostAPIInventoryParams) middleware.Responder {
+	apiInfo := &_database.APIInfo{
+		Type: params.Body.APIType,
+		Name: params.Body.Name,
+		Port: params.Body.Port,
+	}
+	s.dbHandler.APIInventoryTable().CreateAPIInfo(apiInfo)
```

Comment:
maybe we should not ignore the error if it's data that we are getting from the user.

Reply:
What if that the api already exist, isn't it better to use `FirstOrCreate`? Or to verify that it does not exits (name+port+type) before creating a new one?

Reply:
I like the idea with FirstOrCreate more, the apis should be idempotent

Reply:
done